app/com/arpnetworking/metrics/portal/reports/RenderedReport.java

```diff
@@ -40,7 +40,7 @@
      *
      * @return The bytes.
      */
-    InputStream getBytes();
```

Comment:
`ByteSource` is a more natural interface here than `InputStream`, I think; I just didn't know about it when I was initially writing this interface.